misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/launchrequest/LauncherImpl.java

```diff
@@ -90,7 +90,7 @@
 
         command.add(balFile);
 
-        boolean networkLogs = args.get("networkLogs") != null && Boolean.parseBoolean((String) args.get("networkLogs"));
+        boolean networkLogs = args.get("networkLogs") != null && (boolean) args.get("networkLogs");
```

Comment:
Please avoid this unsafe casting of client args (`object` -> `boolean`)

Reply:
Fixed